Generator/Templates/GObject/class.sbntxt

```diff
@@ -38,65 +38,58 @@ namespace {{ namespace }}
     
     public partial class {{ name }} {{ write_inheritance }}
     {
-{{if !fundamental}}
+{{~ if !fundamental ~}}
         private static readonly TypeDescriptor GTypeDescriptor = TypeDescriptor.For("{{ type_name }}", {{ name }}.Native.get_type);
 
         #region Signals
         
-        {{- if signals_have_event_args }}
-        
+        {{~ if signals_have_event_args ~}}
```

Comment:
An empty line is generated when this condition is false.

Reply:
Looks good for me?

```C#
public partial class ParamSpec 
{
    #region Native
        
    protected static class Native
```

Do we have a problem with line breaks between windows / linux?

Reply:
The problem is under the `#region Signals` line, when there is no signal using `EventArgs` the output is like:

```cs
#region Signals


#region Signal1

...

#endregion

#endregion
```

Reply:
Done